README.md

```diff
@@ -128,3 +129,16 @@ Add the following line to your ~/.emacs file to load this mode for files ending
 ;; Scilla mode
 (load-file "/path/to/scilla.el")
 ```
+### Vim plugin for Scilla
+
+You can install the vim config files through Pathogen by:
+```
+git clone https://github.com/edisonljh/vim-scilla.git ~/.vim/bundle/vim-scilla
```

Comment:
Is there a reason why you want to keep it under a separate repo? Would you mind putting all the files here directly in `/misc/?